src/network/core/tcp_game.h

```diff
@@ -38,7 +38,7 @@ enum PacketGameType {
 	PACKET_CLIENT_JOIN,                  ///< The client telling the server it wants to join.
 	PACKET_SERVER_ERROR,                 ///< Server sending an error message to the client.
 
-	/* Packets used for the pre-game lobby. */
+	/* Packets used for the pre-game lobby (unused, but remain for backward/forward compatibility). */
 	PACKET_CLIENT_COMPANY_INFO,          ///< Request information about all companies.
```

Comment:
maybe add a "_UNUSED" to the constants themself?

Reply:
To give that change a bit more visibility, I did that in https://github.com/OpenTTD/OpenTTD/pull/9475 .